mne/viz/_pg_figure.py

```diff
@@ -0,0 +1,2409 @@
+# -*- coding: utf-8 -*-
+"""Base classes and functions for 2D browser backends."""
+
+# Authors: Martin Schulz <dev@earthman-music.de>
+#
+# License: Simplified BSD
+
+import datetime
+import math
+import platform
+from functools import partial
+
+import numpy as np
+from PyQt5.QtCore import (QEvent, QPointF, Qt, pyqtSignal, QRunnable,
+                          QObject, QThreadPool, QRectF)
+from PyQt5.QtGui import (QFont, QIcon, QPixmap, QTransform,
+                         QMouseEvent, QPainter, QImage, QPen)
+from PyQt5.QtTest import QTest
+from PyQt5.QtWidgets import (QAction, QColorDialog, QComboBox, QDialog,
+                             QDockWidget, QDoubleSpinBox, QFormLayout,
+                             QGridLayout, QHBoxLayout, QInputDialog,
+                             QLabel, QMainWindow, QMessageBox,
+                             QPushButton, QScrollBar, QSizePolicy,
+                             QWidget, QStyleOptionSlider, QStyle,
+                             QApplication, QGraphicsView, QProgressBar,
+                             QVBoxLayout, QLineEdit, QCheckBox, QScrollArea)
+from pytestqt.exceptions import capture_exceptions
```

Comment:
pytestqt is not just for testing? as done here it must be installed for everyone.

Reply:
Oh right. But it is only used for testing. I will create a dummy to make it an optional dependency we could then put inside `requirements_testing.txt`